model-optimizer/mo/utils/custom_replacement_config.py

```diff
@@ -394,3 +390,30 @@ def generate_pattern_for_node(graph: Graph, sub_graph_pattern: str, node_name: s
 
     raise RuntimeError('The pattern that uniquely identifies node "{}" using sub-graph pattern "{}" has not been found'.
                        format(node_name, sub_graph_pattern))
+
+
+def load_and_validate_json_config(config_file_name: str):
+    """
+    Reads and validate custom replacement configuration file config_file_name.
+    :param config_file_name: name of the file to read from.
+    :return: The dictionary which was serialize from json config file.
```

Comment:
grammar mistake: `The dictionary which was serialized from json config file.`
Propose to re-write to: `A dictionary serialized from json config file.`